rust/src/main.rs

```diff
@@ -0,0 +1,16 @@
+extern crate reqwest;
+extern crate atom_syndication;
```

Comment:
Is there a way to extend the atom_syndication crate? Otherwise what happens to the following extra fields per/ entry in the feed:

	// atom entry extensions (TODO all optional?) is there no spec for the entry extensions?
	// TODO research why they did not use Atom Extension for the additional fields and if this is kosher
	Event     string         `xml:"event,omitempty"`     // Event - The text denoting the type of the subject event in the alert entry.
	Effective shared.TimeStr `xml:"effective,omitempty"` // Effective - The effective date and time of the information in the alert entry.
	Expires   shared.TimeStr `xml:"expires,omitempty"`   // Expires - The expiry date and time of the information in the alert entry.
	Status    string         `xml:"status,omitempty"`    // Status - The code denoting the appropriate handling of the alert message.
	MsgType   string         `xml:"msgType,omitempty"`   // MsgType - The code denoting the nature of the alert message.
	Urgency   string         `xml:"urgency,omitempty"`   // Urgency - Urgency of the subject event of the alert entry.
	Severity  string         `xml:"severity,omitempty"`  // Severity - Severity of the subject event of the alert entry.
	Certainty string         `xml:"certainty,omitempty"` // Certainty - Certainty of the subject event of the alert entry.
	AreaDesc  string         `xml:"areaDesc,omitempty"`  // AreadDesc - The text describing the affected area of the alert entry.
	Polygon   []string       `xml:"polygon,omitempty"`   // Polygon - The paired values of points defining a polygon that delineates the affected area of the alert entry.
	Circle    []string       `xml:"circle,omitempty"`    // Circle - Note: in the xsd but not explained in CAP 1.2 documentation
	Geocode   Geocode        `xml:"geocode,omitempty"`   // Geocode - The geographic code delineating the affected area of the alert message.
	Parameter []NamedValue   `xml:"parameter,omitempty"` // Parameter - Denotes additional information associated with the

Reply:
If there's a way to get to the extra fields ^ that the CAP spec hacked into their custom atom feeds then we can certainly use this and would just need a way to get and set the extra fields. Looks like feed and entry already have most of the getters and setters.

Reply:
SGTM, I'll look into parsing details next.